<?xml version="1.0"?>
<robot name="guiding_station">
 <link name="gs_base_link">
    <visual>
        <origin xyz="0 0 0.2" rpy="0 0 0"/>
        <geometry>
            <box size="0.4 0.4 0.4"/>
        </geometry>
    </visual>

    <collision>
        <origin xyz="0 0 0.2" rpy="0 0 0"/>
        <geometry>
            <box size="0.4 0.4 0.4"/>
        </geometry>
    </collision>

    <inertial>
        <origin xyz="0 0 0.2" rpy="0 0 0"/>
        <mass value="100.0"/>
         <inertia
            ixx="4.4744" ixy="0.03098" ixz="0.003647"
            iyy="7.1624" iyz="0.1228"
            izz="4.6155"/>
    </inertial>
</link>

 <link name="gimbal_yaw_link">
    <visual>
    <origin xyz="0 0 0" rpy="0 0 0"/>
        <geometry>
            <cylinder length="0.55" radius="0.1"/>
        </geometry>
    </visual>
</link>

<joint name="gimbal_yaw_joint" type="fixed">
    <origin xyz="0 0 0.5" rpy="0 0 0"/>
    <parent link="gs_base_link"/>
    <child link="gimbal_yaw_link"/>
</joint>


<joint name="LED_joint_R" type="fixed">
    <origin xyz="0 0 0" rpy="0 0 0"/>
    <parent link="gimbal_yaw_link"/>
    <child link="gimbal_LED_RED_link"/>
</joint>
<joint name="LED_joint_G" type="fixed">
    <origin xyz="0 0 0" rpy="0 0 0"/>
    <parent link="gimbal_yaw_link"/>
    <child link="gimbal_LED_GREEN_link"/>
</joint>
 <link name="gimbal_LED_RED_link">
    <visual>
    <origin xyz="0 0 0.6" rpy="0 0 0"/>
        <geometry>
            <cylinder length="0.1" radius="0.05"/>
        </geometry>
    </visual>
</link>
 <link name="gimbal_LED_GREEN_link">
    <visual>
    <origin xyz="0 0 0.7" rpy="0 0 0"/>
        <geometry>
            <cylinder length="0.1" radius="0.05"/>
        </geometry>
    </visual>
</link>

<link name="gimbal_pitch_link">
    <visual>
    <origin xyz="0 0 0" rpy="0 1.51 0"/>
        <geometry>
            <cylinder length="0.1" radius="0.05"/>
        </geometry>
      <material name="red">
        <color rgba="0.9 0.1 0.1 1"/>
      </material>
    </visual>
</link>
<joint name="gimbal_pitch_joint" type="fixed">
    <origin xyz="0 0 0.35" rpy="0 0 0"/>
    <parent link="gimbal_yaw_link"/>
    <child link="gimbal_pitch_link"/>
</joint>

<gazebo reference="gs_base_link">
<material>Gazebo/Black</material>
</gazebo>
<gazebo reference="gimbal_LED_RED_link">
<material>Gazebo/Red</material>
</gazebo>
<gazebo reference="gimbal_LED_GREEN_link">
<material>Gazebo/Green</material>
</gazebo>
<gazebo reference="gs_base_link">
<material>Gazebo/Black</material>
</gazebo>

</robot>
